src/metabase/events/schema.clj

```diff
@@ -21,26 +22,31 @@
       view-only      (mc/schema
                       [:map {:closed true}
                        [:user-id [:maybe pos-int?]]
-                       [:object [:fn #(t2/instance-of? :model/Dashboard %)]]])
+                       [:object-id [:maybe pos-int?]]])
       with-dashcards (mut/assoc default-schema
                                 :dashcards [:sequential [:map [:id pos-int?]]])]
   (def ^:private dashboard-events-schemas
-    {:event/dashboard-read             view-only
-     :event/dashboard-create           default-schema
-     :event/dashboard-update           default-schema
-     :event/dashboard-delete           default-schema
-     :event/dashboard-remove-cards     with-dashcards
-     :event/dashboard-add-cards        with-dashcards}))
+    {:event/dashboard-read         view-only
+     :event/dashboard-create       default-schema
+     :event/dashboard-update       default-schema
+     :event/dashboard-delete       default-schema
+     :event/dashboard-remove-cards with-dashcards
+     :event/dashboard-add-cards    with-dashcards}))
 
 ;; card events
 
-(let [default-schema (mc/schema
+(let [view-only      (mc/schema
+                      [:map {:closed true}
+                       [:context ::lib.schema.info/context]
```

Comment:
This matches the `context` column in `query_execution`.
https://github.com/metabase/metabase/blob/c0237533cbe8bae58a0127bacaac40efd16505a9/src/metabase/models/query_execution.clj#L30-L33